animation_nodes/math/rotation_conversion.pyx

```diff
@@ -1,6 +1,14 @@
-from libc.math cimport atan2, sqrt, hypot, cos, sin
+cimport cython
+from libc.math cimport (
+    M_PI as PI, sqrt, hypot, abs, sin, cos, asin,
+    acos, atan2, copysign
+)
 from . matrix cimport normalizeMatrix_3x3_Part
 from . conversion cimport toPyEuler3
+from . vector cimport lengthVec3
+from . quaternion cimport setUnitQuaternion
+from mathutils import Vector, Matrix, Euler
+from .. math cimport quaternionNormalize_InPlace
```

Comment:
Those are still not ordered. Longer lines at the button. And maybe put `from libc.math cimport` at the very end.